frontend/src/integrations/gerrit/config.js

```diff
@@ -0,0 +1,39 @@
+import config from '@/config';
+import GerritConnect from './components/gerrit-connect.vue';
+
```

Comment:
You will also need to add an `svg` gerrit icon to `crowd-icons.svg`. Without it, some of the default reports might have the icons broken when it comes to Gerrit's activities.
Icon should be 16x16

Reply:
I searched for a svg icon, but couldn't find

Reply:
Hey @khalifapro, here's good resolution assets for both png and svg:
![gerrit](https://github.com/CrowdDotDev/crowd.dev/assets/20134207/20198da4-15fe-4113-b099-da604b4813e8)
![gerrit](https://github.com/CrowdDotDev/crowd.dev/assets/20134207/449595a1-1d17-47b0-82c4-d51e8de54e0a)
[gerrit-images.zip](https://github.com/CrowdDotDev/crowd.dev/files/13715170/gerrit-images.zip)

Reply:
Done